config/locales/3ilm.ar.yml

```
@@ -0,0 +1,55 @@
+    stats:
+      title: "إحصائيات الموقع"
+      lesson: "درس"
+      lecture: "محاضرة"
+      book: "كتاب"
+      series: "سلسلة"
+      last_updated: "آخر تحديث: %{date}"
+
+    # Index Page Sections
+    index:
+      recent_lessons: "جديد الدروس"
+      recent_lectures: "جديد المحاضرات" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)